sdk/python/feast/offline_store.py

```diff
@@ -612,3 +558,10 @@ def _get_requested_feature_views_to_features_dict(
 {% endfor %}
 ORDER BY event_timestamp
 """
+
+
+def get_offline_store(config: RepoConfig) -> Type[OfflineStore]:
+    if config.provider == "gcp":
```

Comment:
What should the user select if they want to pull data from BigQuery and write to a local store?